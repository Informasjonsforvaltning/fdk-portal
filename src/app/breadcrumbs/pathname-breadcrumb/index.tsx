import React, { FC } from 'react';
import localization from '../../../lib/localization';

interface Props {
  pathName?: string;
}

const PathNameBreadcrumb: FC<Props> = ({ pathName }) =>
  pathName ? <span>{localization.menu[pathName]}</span> : null;

export default PathNameBreadcrumb;
