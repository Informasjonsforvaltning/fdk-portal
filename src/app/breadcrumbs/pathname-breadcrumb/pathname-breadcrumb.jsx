import React from 'react';
import localization from '../../../lib/localization';

export const PathNameBreadcrumb = ({ pathName }) => (
  <span>{localization.menu[pathName]}</span>
);
