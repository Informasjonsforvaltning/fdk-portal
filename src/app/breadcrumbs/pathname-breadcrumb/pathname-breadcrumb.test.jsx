import React from 'react';
import { shallow } from 'enzyme';
import { PathNameBreadcrumb } from './pathname-breadcrumb';

test('should render PathNameBreadcrumb  correctly', () => {
  const defaultProps = {
    pathname: 'about'
  };
  const wrapper = shallow(<PathNameBreadcrumb {...defaultProps} />);
  expect(wrapper).toMatchSnapshot();
});

test('should render PathNameBreadcrumb correctly with no pathname', () => {
  const defaultProps = {};
  const wrapper = shallow(<PathNameBreadcrumb {...defaultProps} />);
  expect(wrapper).toMatchSnapshot();
});
