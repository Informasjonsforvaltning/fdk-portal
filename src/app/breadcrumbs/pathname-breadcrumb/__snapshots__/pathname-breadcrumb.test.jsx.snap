// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PathNameBreadcrumb  correctly 1`] = `<span />`;

exports[`should render PathNameBreadcrumb correctly with no pathname 1`] = `<span />`;
